app/controllers/sessions_controller.rb

```diff
@@ -107,11 +107,9 @@ def create
           redirect_to profile_path,
                       alert: "That way to log in cannot be added because it is associated to an email address that is already in use!" # TODO i18n
         else
-          Rails.logger.fatal "IllegalState: OAuth data: #{request.env['omniauth.auth']}; " \
-                             "status: #{@handler_result.outputs[:status]}"
-          raise IllegalState, "SessionsCreate errors: #{@handler_result.errors.inspect
-                              }; Last exception: #{$!.inspect}; Exception backtrace: #{$@.inspect
-                              }; status: #{@handler_result.outputs[:status]}"
+          Rails.logger.fatal "IllegalState: OAuth data: #{request.env['omniauth.auth']
+                             }; status: #{@handler_result.outputs[:status]}"
```

Comment:
Won't this result in weird spacing and a line break in the message?  

```
[1] pry(main)> Rails.logger.fatal "hi 
[1] pry(main)*         there"
hi 
        there
```

Maybe the original just needed some parens around the string?